Tasks/Npm/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -3,14 +3,15 @@
   "loc.helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkID=613746)",
   "loc.description": "Run a npm command",
   "loc.instanceNameFormat": "npm $(command)",
-  "loc.group.displayName.advanced": "Advanced",
-  "loc.input.label.command": "Command",
+  "loc.input.label.cwd": "Working Directory",
+  "loc.input.help.cwd": "Current working directory where npm command is run.  Defaults to the root of the repo.",
```

Comment:
Remove the word "Current".
"where _the_ npm"
Remove the double space after the "."